packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -104,7 +104,11 @@ export function getBeaconPoolApi({
           try {
             await validateBlsToExecutionChange(chain, blsToExecutionChange);
             chain.opPool.insertBlsToExecutionChange(blsToExecutionChange);
-            await network.gossip.publishBlsToExecutionChange(blsToExecutionChange);
+            if (chain.clock.currentEpoch >= chain.config.CAPELLA_FORK_EPOCH) {
+              await network.gossip.publishBlsToExecutionChange(blsToExecutionChange);
+            } else {
+              await chain.cacheBlsToExecutionChanges(blsToExecutionChange);
```

Comment:
Why is it necessary to explicitly call this? `chain.opPool.insertBlsToExecutionChange` should eventually write to DB on shutdown